tensorboard/examples/plugins/example_basic/smoke_test.sh

```diff
@@ -66,7 +66,8 @@ curl -fs "http://localhost:${port}/data/plugin/example_basic/index.js" >index.js
 diff -u example-plugin/tensorboard_plugin_example/static/index.js index.js
 curl -fs "http://localhost:${port}/data/plugins_listing" >plugins_listing
 cat plugins_listing; printf '\n'
-grep -qP '"example_basic":(?:(?!"enabled").)*+"enabled": *true' plugins_listing
+
+grep -q '"example_basic": {[^}]*"enabled": true' plugins_listing
```

Comment:
This JSON response contains nested fields, like

```json
{
    "example_basic": {
        "loading_mechanism": {
            "type": "IFRAME",
            "module_path": "/data/plugin/example/static/index.js"
        },
        "enabled": true
    }
}
```

and there is no guarantee that the `"enabled": true` needle comes before
the closing brace for the `loading_mechanism`. In particular, we do not
pass `sort_keys=True` to `json.dumps` in `http_util.Respond`.

Presuming that the issue is that you use [a newer version of macOS,
which broke `grep -P`][1], please feel free to replace this with the
equivalent `perl` in a separate commit, but this PR should just be a
move. As a workaround, you can install GNU `grep` or downgrade to a BSD
`grep` with `-P`.

[1]: https://stackoverflow.com/q/16658333

Reply:
Ah I see, thanks.  It so happens to come out in a passing order for me (`loading_mechanism` after `enabled`, with nothing else nested anyway), which is how I was able to overlook this :)  Reverted.

FWIW the only reason I was concerned about this at all was that the test failed in CI and I wanted to debug it locally.  The real reason for the CI failure was just a timeout, nothing to do with the grep obviously.  Still we may need to address that too if flakiness is an issue.